### ISSUE TITLE
Angular 9 upgrade?

### ISSUE THREAD

**Issue by senzacionale** [why520crazy/ngx-validator#29]
Hi,
I am getting this warnings when I am using angular 9

```
npm WARN @why520crazy/ngx-validator@1.0.0 requires a peer of @angular/common@^7.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN @why520crazy/ngx-validator@1.0.0 requires a peer of @angular/forms@^7.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN @why520crazy/ngx-validator@1.0.0 requires a peer of @angular/core@^7.0.0 but none is installed. You must install peer dependencies yourself.
```

Can be a problem?

**Comment by why520crazy** [comment]
please bump @why520crazy/ngx-validator to 9.0.0 try again
